### PR TITLE
Fix/407 jvm locale in properties

### LINKED ISSUE #407: Allow JVM locale to be set via jbake.properties
See #182

### PR DESCRIPTION
**Pull request by rustamzh**
Attempt to close #407. I have compared it with specifying JAVA_OPTS inside gradlew - no difference in the test.  Welcome any reviews.

### CODE REVIEW

File: jbake-core/src/main/java/org/jbake/launcher/Main.java [hunk 1]
```diff
@@ -177,4 +176,12 @@ private void initStructure(CompositeConfiguration config, String type, String so
 		}
 	}
 
+	private void setupLocale(CompositeConfiguration config){
+		if (config.containsKey("jvm.language") && config.containsKey("jvm.country")){
+			String language = config.getString("jvm.language");
```
Comment (jonbullock) [inline jbake-core/src/main/java/org/jbake/launcher/Main.java]: Could you convert these config strings to constants like here: https://github.com/jbake-org/jbake/blob/master/jbake-core/src/main/java/org/jbake/app/ConfigUtil.java#L28

File: jbake-core/src/main/java/org/jbake/launcher/Main.java [hunk 2]
```diff
@@ -177,4 +176,12 @@ private void initStructure(CompositeConfiguration config, String type, String so
...
+			String country = config.getString("jvm.country");
+			Locale newLocale = new Locale(language, country);
```
Comment (jonbullock) [inline jbake-core/src/main/java/org/jbake/launcher/Main.java]: You do have access to [LocaleUtils](https://commons.apache.org/proper/commons-lang/javadocs/api-3.7/org/apache/commons/lang3/LocaleUtils.html) and can be used like this if you want: https://github.com/jbake-org/jbake/blob/master/jbake-core/src/main/java/org/jbake/template/ThymeleafTemplateEngine.java#L63

**Comment by rustamzh** [comment]
Thank you for the review. I have revised implementation.

**Comment by ancho** [comment]
Hi. Thanks. Maybe we should setup the locale in the Oven bake method? The gradle plugin does not make use of the Main class.

**Comment by rustamzh** [comment]
Hello. Moved locale setup to the Oven.

**Comment by rustamzh** [comment]
Sorry for closing PR. Thought it will close commenting box.

**Pull request merged**